moboff.py

```diff
@@ -39,7 +39,7 @@ def cli():
 @cli.command('download', short_help='Downloads and sends the video')
 @click.option('--link', prompt='Video/Playlist link to download',
               help='Paste the link here to download and send to your device')
-@click.option('--newdevice', help='Overwrite the saved device')
+@click.option('--newdevice', help='Overwrite the saved device for particular download command')
```

Comment:
This looks a bit unnecessary. Flags *are* meant to be a command instance specific.